pkg/vetter/mtlsprobes/vet.go

```diff
@@ -185,12 +190,19 @@ func (m *MtlsProbes) Vet() ([]*apiv1.Note, error) {
 						glog.Errorln("Probe port number is out of range, skipping to next pod")
 						continue
 					} else {
+
 						// get endpoint for the pod
 						podEndpoint, err := getPodEndpoint(endpointsList, p)
 						if err != nil {
 							glog.Errorln("Error getting pod endpoint, skipping to next pod")
 							continue
 						}
+						// Extracts the statusPort from the config.
+						statusPort := util.ProxyStatusPort(c)
+						if statusPort == probePortNum {
+							glog.Infof("Skipping mTLS check as the sidecar status port is used.")
```

Comment:
should be Debug in my opinion as it will be the normal case

Reply:
I just removed the additional logging... I really added it for initial debugging, but it is not really needed.